### PR TITLE
Metronome networking documentation

### PR DESCRIPTION
**Pull request by timcharper**
## Jira Ticket
Before creating this pull request, please make sure you have a separate ticket for the docs team to track their work to review and merge, even for minor edits. If you need assistance, ask in the #doc-team channel on slack.

https://jira.d2iq.com/browse/D2IQ-68135

## Description of changes being made

We add documentation for Metronome networking. Metronome networking applies for DC/OS 2.0.4 and 2.1.0.

## Checklist
- [x] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [x] Test all commands and procedures, if applicable.
- [x] Create your PR against `staging`, not `master`. 
- [ ] Provide an estimated date for deploying the doc change. Note: Improvements or fixes can be merged ASAP. 
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

**Comment by cmastr** [comment]
cannot merge until 2.0.4 patch is ready to release.

File: pages/mesosphere/dcos/2.0/deploying-jobs/examples/index.md [hunk 1]
```diff
@@ -214,3 +214,28 @@ The jobs groups are then assigned permissions to users `Cory` and `Alice` to res
     ![Alice job view](/mesosphere/dcos/2.0/img/GUI-Restricted_User-Jobs_View_Alice-1_12.png)
 
     Restricted view for 'Alice'
+
+# Creating a job with joins a container network
```
[the remark this answers is not in the document]
Comment (cmastr) [reply pages/mesosphere/dcos/2.0/deploying-jobs/examples/index.md:218]: or "with joins to a container network". Need clarification from @timcharper

Comment (timcharper) [inline pages/mesosphere/dcos/2.0/deploying-jobs/examples/index.md:218]:
```suggestion
# Creating a job which joins a container network
```

Comment (timcharper) [reply pages/mesosphere/dcos/2.0/deploying-jobs/examples/index.md:218]: which joins is correct, thank you

File: pages/mesosphere/dcos/2.1/deploying-jobs/examples/index.md [hunk 2]
```diff
@@ -214,3 +214,26 @@ The jobs groups are then assigned permissions to users `Cory` and `Alice` to res
     ![Alice job view](/mesosphere/dcos/2.1/img/GUI-Restricted_User-Jobs_View_Alice-1_12.png)
 
     Figure 8. Restricted view for 'Alice'
+
+# Creating a job with joins a container network
```
Comment (timcharper) [inline pages/mesosphere/dcos/2.1/deploying-jobs/examples/index.md:218]:
```suggestion
# Creating a job which joins a container network
```

**Pull request merged**